mungegithub/features/repo-updates.go

```diff
@@ -36,16 +36,15 @@ import (
 )
 
 const (
-	ownerFilename = "OWNERS" // file which contains owners and assignees
+	ownerFilename = "OWNERS" // file which contains approvers and reviewers
 	// RepoFeatureName is how mungers should indicate this is required
 	RepoFeatureName = "gitrepos"
 )
 
 type assignmentConfig struct {
 	Assignees []string `json:assignees yaml:assignees`
```

Comment:
What is this being used for now? Do we need to update/rationalize the "top of .md file assignment" stuff to this new nomenclature?